sympy/combinatorics/tests/test_fp_groups.py

```diff
@@ -125,6 +125,413 @@ def test_coset_enumeration():
     # this test function contains the combined tests for the two strategies
     # i.e. HLT and Felsch strategies.
 
+    # Test for Issue #11352
+    F, a, b, c = free_group("a, b, c")
+    f = FpGroup(F, [a**11, b**5, c**4, (a*c)**3, b**2*c**-1*b**-1*c, a**4*b**-1*a**-1*b])
+    H = [c, b, c**2]
+    C_r = coset_enumeration_c(f, H)
+    C_r.compress(); C_r.standardize()
```

Comment:
Remove this line. And the line above this i.e remove lines [132](https://github.com/sympy/sympy/pull/11356/files#diff-07f5c9eafa6f27616138278a54dd0e75R132) and [133](https://github.com/sympy/sympy/pull/11356/files#diff-07f5c9eafa6f27616138278a54dd0e75R133) . Sine testing is to be there only for `coset_enumeration_c`.